xilem/src/driver.rs

```diff
@@ -1,17 +1,70 @@
 // Copyright 2024 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
-use masonry::{app_driver::AppDriver, widget::RootWidget};
-use xilem_core::MessageResult;
+use std::sync::Arc;
+
+use masonry::{
+    app_driver::AppDriver,
+    event_loop_runner::{self, EventLoopProxy, MasonryUserEvent},
+    widget::RootWidget,
+    WidgetId,
+};
+use xilem_core::{DynMessage, Message, MessageResult, ProxyError, RawProxy, ViewId};
 
 use crate::{ViewCtx, WidgetView};
 
 pub struct MasonryDriver<State, Logic, View, ViewState> {
     pub(crate) state: State,
     pub(crate) logic: Logic,
     pub(crate) current_view: View,
-    pub(crate) view_ctx: ViewCtx,
+    pub(crate) ctx: ViewCtx,
     pub(crate) view_state: ViewState,
+    #[allow(unused)]
+    // Keep the runtime alive
+    // Is it more idiomatic to stick this in the `ViewCtx`?
```

Comment:
The question I'm raising here is whether the context needs direct access to the runtime - according to tokio's docs, there is no difference between `Handle` and `&Runtime`

Reply:
Yeah good point, I mean I don't think it's too important then. But I guess we can still avoid an additional attribute. I mean we can still put it back to the driver in case we need the `Runtime` there for whatever reason...